src/ihela_client/merchant_client.py

```
@@ -132,60 +132,82 @@ def customer_lookup(self, bank_slug, customer_id=None, account_number=None):
+                "merchant_description": merchant_description or description,
                 "merchant_reference": reference,
-                "user": user,
-                "bank": bank,
-                "bank_client_id": bank_client_id,
                 "redirect_uri": redirect_uri,
+                "payment_product_id": payment_product_id,
+                "pin_code": pin_code,
             }
+            bill_data = {k: v for k, v in bill_data.items() if v is not None}
             url = iHela_ENDPOINTS["BILL_INIT"]
             bill_ = requests.post(
```

Ruff (0.15.21)

[error] Probable use of `requests` call without timeout

(S113)